packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -14,18 +14,30 @@ interface FacetOptions {
 }
 
 describe('Facet Test Suite', () => {
+  const pageUrl = `${Cypress.env('examplesUrl')}/s/quantic-facet`;
```

Comment:
do you think we should have a const for `${Cypress.env('examplesUrl')}/s`
as I think we will reuse this for other component test?

Reply:
Yes, other test suites will also need to build a page URL like this. We could even manage to make it the base URL, so test suites could simply do:

    cy.visit('s/quantic-facet');
    cy.visit('s/quantic-category-facet');
    ... and so on